src/core/Modal/BaseModal/BaseModal.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { ThemedBackground } from '../../../utils/storybook';
+import BaseModal from '../BaseModal/BaseModal';
+
+storiesOf('Modal/Base Modal', module)
+  // Litmus Portal
+  .add('Litmus Portal', () => (
+    <ThemedBackground platform="litmus-portal">
+      <BaseModal
+        isOpen={true}
+        handleClose={() => console.log('clicked')}
+        aria-labelledby="simple-modal-title"
+        aria-describedby="simple-modal-description"
+        hasCloseBtn
+      >
+        Base Modal
```

Comment:
We can keep this only as `Modal`

Reply:
@S-ayanide create a new name because the material-UI modal and local modal names were conflicting so tied to modify the same.